test/test_attributes.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright (c) 2025 National Land Survey of Finland (Maanmittauslaitos)
+#
+#  This file is part of geogen-algorithms.
+#
+#  This source code is licensed under the MIT license found in the
+#  LICENSE file in the root directory of this source tree.
+
+import geopandas as gpd
+import pytest
+from shapely.geometry import Polygon
+
+from geogenalg import attributes
+
+
+@pytest.mark.parametrize(
+    ("source_gdf", "target_gdf", "expected_attribute_value"),
```

Comment:
I think it would be good idea to have "expected_geometry" here and then the corresponding `assert geometry ==...` in the actual test.